ruby.c

```diff
@@ -271,6 +274,27 @@ usage(const char *name, int help)
 	SHOW(features[i]);
 }
 
+static void
+usage_dump(const char *name)
+{
+    struct message {
+	const char *str;
+	unsigned short namelen, secondlen;
+    };
+    static const struct message usage_msg[] = {
+	M("--dump=insns",     "", "dump YARV instructions"),
```

Comment:
Nice patch 🎉 

I feel "dump instruction sequences" is better. Because the name "YARV" is not exposed to public.

Reply:
okay